cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -100,70 +108,92 @@ pub(crate) enum Inst {
         dst: Writable<Reg>,
     },
 
-    /// cmp (b w l q) (reg addr imm) reg
+    /// Integer comparisons: cmp (b w l q) (reg addr imm) reg.
```

Comment:
Random comment: if at some point you add `tst` as an instruction, it has the same "shape" as `cmp` and so would fit naturally here.  (cmp = subtract operands, set flags, throw away result; tst = AND operands; set flags; throw away result).